net/tailscale/Makefile

```diff
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=tailscale
-PKG_VERSION:=1.24.2
+PKG_VERSION:=1.32.0
```

Comment:
would be ideal to bump to 1.32.3 for the fixes to the CVEs https://github.com/tailscale/tailscale/releases/tag/v1.32.3